src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -525,6 +525,14 @@ internal override bool GenerateDebugInfo
             }
         }
 
+        public sealed override bool AreLocalsZeroed
+        {
+            get
+            {
+                return _property.AreLocalsZeroed && base.AreLocalsZeroed;
```

Comment:
>return _property.AreLocalsZeroed && base.AreLocalsZeroed; [](start = 16, length = 57)

It looks like this will be checking containing types twice. First, _property.AreLocalsZeroed will check types, then base.AreLocalsZeroed will check the same types. Perhaps SourcePropertySymbol shouldn't check types (we can rename the property to reflect this fact)?

Reply:
Maybe we can call it `HasSkipLocalsInitAttribute` as well. I'll make a new PR today for modules and update it there.

Reply:
Fixed it in #25732